app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,9 @@
+class ProjectsController < ApplicationController
+  def show
+    @project = Project.find(params[:id])
+    @project_material = @project.material
+    @project_challenge_theme = @project.challenge.theme
+    @contestant_count = @project.contestant_count
+    @average_years_exp = @project.average_years_exp
+  end
```

Comment:
Careful here. We usually only want to send one single instance variable to our views. We can then call all these methods from the view, so we'd only want to have `@project = Project.find(params[:id])` here. We want to keep these controllers thin